src/components/Table/Table.stories.jsx

```diff
@@ -62,7 +63,23 @@ export const Basic = () => (
 export const BasicReactTable = () => {
     const data = React.useMemo(() => tableDataFlat, []);
     const columns = React.useMemo(() => [
-        {Header: 'Name', id: 'name', accessor: row => row.name.value},
+        {
+            Header: 'Name',
+            id: 'name',
+            accessor: row => row.name.value,
+            Cell: cellInfo => {
+                const {row} = cellInfo;
+                return (
+                    <ListItem
+                        iconSize="default"
+                        typographyVariant="body"
+                        iconStart={row.original.name.icon}
+                        label={row.values.name}
+                    />
```

Comment:
As the `ListItem` component render a `li` we can't use it here because of the semantic.